node/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -508,7 +508,7 @@ cancelApplyingPtxs =
 cancelSpecificApplyingPtx :: TxId -> Update ()
 cancelSpecificApplyingPtx txId =
     wsWalletInfos . traversed .
-    wsPendingTxs . ix txId %= cancelApplyingPtx
+    wsPendingTxs . at txId .= Nothing
```

Comment:
Looks good, these transactions will remain in history with status `NotTracked` and won't be touched by resubmitter anymore

Reply:
I think we want to omit this change, that we should stick with it being a change of state.

We should add a new endpoint to delete all pending txs in the `PtxWontApply` state. So lets just omit this patch from the PR.